src/test/resources/pieces_for_view.sample

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
we can use the existing sample files already available instead of duplicating them , it will be easier to maintain